src/lib/components/market/item_row_wrapper.ts

```diff
@@ -130,6 +135,23 @@ export class ItemRowWrapper extends FloatElement {
             });
         }
 
+        // Fetch bluegem data if needed
+        if (this.itemInfo && this.asset && isBlueSkin(this.itemInfo)) {
+            try {
+                const bluegemResponse = await ClientSend(FetchBluegem, {
```

Comment:
nit: You can assign `this.bluegemData` directly from the response instead of this intermediate variable.